src/app/components/SearchResultsPage/SearchResultsPage.jsx

```diff
@@ -23,24 +23,28 @@ const SearchResultsPage = (props) => {
     sortBy,
     field,
     spinning,
+    error,
   } = props;
 
   const facetList = facets && facets.itemListElement ? facets.itemListElement : [];
-  const totalHits = searchResults ? searchResults.totalResults : 0;
-  const totalPages = Math.floor(totalHits / 50) + 1;
+  const totalHits = searchResults ? searchResults.totalResults : undefined;
+  const totalPages = totalHits ? Math.floor(totalHits / 50) + 1 : 0;
   const results = searchResults ? searchResults.itemListElement : [];
   const breadcrumbs = (
     <Breadcrumbs query={searchKeywords} type="search" />
   );
   const createAPIQuery = basicQuery(props);
+  const h1searchKeywordsLabel = searchKeywords ? `for ${searchKeywords}` : '';
+  const h1pageLabel = totalPages ? `page ${page} of ${totalPages}` : '';
+  const h1Label = `Search results ${h1searchKeywordsLabel} ${h1pageLabel}`;
 
   return (
     <DocumentTitle title={`${searchKeywords ? searchKeywords + ' | ' : ''} Search Results | Research Catalog | NYPL`}>
       <main className="main-page">
         <div className="nypl-page-header">
           <div className="nypl-full-width-wrapper">
             {breadcrumbs}
-            <h1 aria-label={`Search results for ${searchKeywords} page ${page} of ${totalPages}`}>
+            <h1 aria-label={h1Label}>
               Search results
```

Comment:
On an error page, the title of the page is still "Search results" so it's fine. You get the error message below that on the page's content.